pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsReaderImpl.java

```diff
@@ -83,8 +83,13 @@ public MultiTopicsReaderImpl(PulsarClientImpl client, ReaderConfigurationData<T>
 
                 @Override
                 public void received(Consumer<T> consumer, Message<T> msg) {
+                    final MessageId messageId = msg.getMessageId();
```

Comment:
Actually, the `TopicMessageImpl` has its own message id. Therefore, it works well. but I think it's better to keep the same behaviour as `ReaderImpl`.